packages/yoastseo/src/scoring/assessments/SCORING SEO.md

```diff
@@ -1,6 +1,21 @@
 # Overview of the SEO assessments scoring criteria
+### How are individual and overall traffic lights assigned?
```

Comment:
I added the `overall` because I think the same logic is used for assigning the overall score? In `scoreToRating.js`, which is called in `ScoreIcon.js`, which is called in `Results.js` which takes the overall score from `seo.score`.